### PR TITLE
Add Exercise 14 files

### PR DESCRIPTION
**Pull request by ElizaAlimasa**
Please review my files, Thank you!

### CODE REVIEW

File: alimasa_eliza/countries_script.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     <p>Capital: ${country.capital ? 
+            country.capital[0] : 'N/A'}</p>
+     <p>Timezones: ${country.timezones ? 
+                          country.timezones.join(', ') : 'N/A'}</p>
+     <p>Region: ${country.region ? 
+                            country.region : 'N/A'}</p>
+    `;
+    document.getElementById('country_details').innerHTML = details;
+    
+    return fetch('https://restcountries.com/v3.1/region/' + country.region);
```
Comment (barrantes-sofiaanne) [inline alimasa_eliza/countries_script.js:33]: Do not create another fetch inside a then method.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2637aafa621_0_10
Click the link for reference

File: alimasa_eliza/countries_script.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+     `;
+    }).join('');
+    document.getElementById('same_region_countries').innerHTML = `
+     <h2>Countries in the Same Region (${region})</h2>
+     <div class="country-list">${sameRegionCountriesList}</div>
+    `;
+    })
+    .catch(function(error) {
+    console.error('Error fetching data:', error);
+    document.getElementById('country_details').innerHTML = '<p>An error occurred: '
```
Comment (barrantes-sofiaanne) [inline alimasa_eliza/countries_script.js]: Exceeds 80 columns

**Review by barrantes-sofiaanne — COMMENTED** [review]
Please read my review

**Comment by ElizaAlimasa** [comment]
I add changes in my files, please review it again. Thank you!

File: alimasa_eliza/script.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    comment_list.appendChild(li);
+  });
+}
+
+function sortComments(order) {
+    if (order === 'asc') {
+      comments.sort((a, b) => new Date(a.timestamp) - new Date(b.timestamp));
+    } else {
+      comments.sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp));
+    }
```
Comment (barrantes-sofiaanne) [inline alimasa_eliza/script.js]: Do not use arrow function as a method of an object.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29a1fb1fdca_2_0
Click this link for reference

**Review by barrantes-sofiaanne — COMMENTED** [review]
I added another review for your js

**Review by barrantes-sofiaanne — APPROVED** [review]
You can now merge it to main

**Comment by ElizaAlimasa** [comment]
Thank you!

**Pull request merged**